app/src/main/java/com/mobileproto/hireddit/hireddit/visuals/SpeechFragment.java

```diff
@@ -76,6 +78,12 @@ public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         if (getArguments() != null) {
         }
+        ShakeDetector.create(this.getContext(), new ShakeDetector.OnShakeListener() {
+            @Override
+            public void OnShake() {
+                shake();
+            }
+        });
```

Comment:
yeah, since the only things we'd pupt are quiet mode and shake, and we might as well just have them as buttons on screen since they won't take much space.